components/terraform/vault-deployment/helm-install.tf

```diff
@@ -37,7 +35,15 @@ resource "helm_release" "vault" {
   ]
 }
 
-
+resource "helm_release" "vault-secrets-operator" {
```

Comment:
installs the first beta release of the secrets operator to the dedicated namespace